addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15124,7 +15124,13 @@ msgctxt "#22083"
 msgid "Play all"
 msgstr ""
 
-#empty strings from id 22084 to 23048
+#. Description of a setting to choose what happens when clicking on an album in the music library on the home screen
```

Comment:
Perhaps the home screen part is a little misleading then, the video side of things uses just 'Default select action'.  However, I deliberately limited where this works for music as otherwise it would totally change the music navigation that everyone is used to.  I took 'home screen' to mean the screen from where one can navigate to other areas (music, video, pvr, settings etc) of Kodi.